### PR TITLE
added revision d03115

### PR DESCRIPTION
**Pull request by semaf**
Raspberry Pi 4 8GB Rev 1.5 Revision d03115 added to `lib/raspberry_pi_helper.py`

### CODE REVIEW

File: bin/install.sh [hunk 1]
```diff
@@ -121,10 +121,10 @@ fi
 if [ -z "${REPOSITORY}" ]; then
   if [ "$WEB_UPGRADE" = false ]; then
     set -x
-    REPOSITORY=${1:-https://github.com/screenly/screenly-ose.git}
+    REPOSITORY=${1:-https://github.com/semaf/screenly-ose.git}
```
Comment (vpetersson) [inline bin/install.sh:124]: Please revert this.

File: bin/install.sh [hunk 2]
```diff
@@ -121,10 +121,10 @@ fi
...
   else
     set -e
-    REPOSITORY=https://github.com/screenly/screenly-ose.git
+    REPOSITORY=https://github.com/semaf/screenly-ose.git
```
Comment (vpetersson) [inline bin/install.sh:127]: Please revert this.

Comment (semaf) [reply bin/install.sh]: Sorry for the issue. Done.

**Review by vpetersson — APPROVED** [review]

**Comment by vpetersson** [comment]
LGTM. Thanks.

**Pull request merged**